src/shelly_switch.cpp

```diff
@@ -25,6 +25,14 @@
 
 namespace shelly {
 
+const HAPUUID kHAPCharacteristic_EveConsumption = {
+    0x52, 0x9F, 0xA2, 0x05, 0x26, 0x9C, 0x27, 0x8F,
+    0xFF, 0x48, 0x9E, 0x07, 0x0D, 0xF1, 0x63, 0xE8};
```

Comment:
please move closing brace to the next line, ditto below.

Reply:
That’s what `make format` does

Reply:
leave a comma at the end and it won't